### PR TITLE
add headless label

### PR DESCRIPTION
**Pull request by kristinapathak**
Resolves https://github.com/open-telemetry/opentelemetry-operator/issues/898

### CODE REVIEW

**Review by jaronoff97 — APPROVED** [review]

**Comment by mingh2** [comment]
Hi, wanted to quickly followup on this, is there an expected timeline when this will be merged and rollout to the newest operator version?

**Comment by VineethReddy02** [comment]
@mingh2 This can be part of the release `0.64.1` the PR is already up and waiting for the final review. Expect the release in the next 2-3 days.

**Review by VineethReddy02 — APPROVED** [review]

**Review by pavolloffay — APPROVED** [review]

**Comment by rbaumgar** [comment]
@VineethReddy02 Today is Feb 5, 2023. I still have Red Hat OpenShift distributed tracing data collection 0.63.1-4 out of redhat operators: registry.redhat.io/redhat/redhat-operator-index:v4.12
OCP 4.12

Any updates?

**Pull request merged**
